src/platform/imx8/imx8.x.in

```diff
@@ -461,7 +461,7 @@ SECTIONS
 
```

Comment:
@paulstelian97 thanks for your contribution. Please use commit instead of PR number. When someone clones the repo the PR might make little sense. Just use the format:

commit 123456789abcd ("commit subject").